app/models/picture.rb

```diff
@@ -0,0 +1,5 @@
+class Picture < ActiveRecord::Base
+  has_attached_file :image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: ":style/monkey.png"
```

Comment:
Remember that you can add a line break after each comma. It shuts Hound up and gives you better readability.